Libs/MRML/Core/vtkMRMLTransformNode.cxx

```diff
@@ -1193,6 +1194,22 @@ void vtkMRMLTransformNode::Inverse()
   this->TransformModified();
 }
 
+//----------------------------------------------------------------------------
+void vtkMRMLTransformNode::InverseName()
```

Comment:
I feel like this should be an independent member variable like `HasBeenInverted` and the display should be just a gui feature.  Although I guess the benefit of changing the name is that it will appear consistently on all comboboxes, list view, etc.  But it will also be in the default filename, which is kind of odd.  Overall I guess it's fine to change the name.

Reply:
This is an interesting point. We actually have a notion of "inverted" transform (shown in the Information section in Transforms module), but it is really about the consistency of the node name and content. If you invert the transform and save it then you do want to reflect that in the name.

Reply:
Yes, makes sense.